rust/telemetry/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "chroma-telemetry"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+async-trait = "0.1"
+downcast-rs = "1.2"
```

Comment:
this crate can be implemented in a very simple way ourselves, can we not implement?